### PR TITLE
Include a section for merge_exclude_columns

### LINKED ISSUE #2197: Document addition of `merge_exclude_columns`
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

https://docs.getdbt.com/docs/building-a-dbt-project/building-models/configuring-incremental-models#strategy-specific-configs

### What part(s) of the page would you like to see updated?

- Add `merge_exclude_columns` in parallel to `merge_include_columns`
- Mention and link from v1.3 migration guide

### Additional information

https://github.com/dbt-labs/dbt-core/pull/5457

### PR DESCRIPTION
**Pull request by dpguthrie**
## What are you changing in this pull request and why?

Adding a section for `merge_exclude_columns` to go with the similar config `merge_update_columns`.  This is functionality that was added [here](https://github.com/dbt-labs/dbt-core/pull/5457)

Closes #2197 

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

**Comment by runleonarun** [comment]
This looks great, @dpguthrie. Does this need to be versioned or is it available in Core 1.0 and higher?

**Comment by dpguthrie** [comment]
Good callout @runleonarun - this is only in 1.3 and later.  How do I specify versions here?

**Comment by dpguthrie** [comment]
@runleonarun I wrapped the entire strategy specific config section in a `VersionBlock`.  Right now it's showing for all 1.* versions and was only added in 1.3.

**Comment by runleonarun** [comment]
Nice catch @dpguthrie! I will review and merge this today!

File: website/docs/docs/build/incremental-models.md [hunk 1]
```diff
@@ -309,6 +311,27 @@ select ...
 
 </File>
 
+Alternatively, specify a list of columns to exclude from being updated by passing a list of column names to a `merge_exclude_columns` config.
```
Comment (runleonarun) [inline website/docs/docs/build/incremental-models.md]:
```suggestion
Alternatively, you can specify a list of columns to exclude from being updated by passing a list of column names to a `merge_exclude_columns` config.
```

**Review by runleonarun — DISMISSED** [review]
Looks good, approving to merge with one tiny wording recommendation.

**Review by runleonarun — APPROVED** [review]
🚢 it!

**Pull request merged**